src/main/java/org/apache/commons/collections4/list/CursorableLinkedList.java

```diff
@@ -57,7 +57,9 @@
  *
  * @see java.util.LinkedList
  * @since 1.0
+ * @deprecated
```

Comment:
All the other deprecated tags in the library have a comment on what to use instead. We currently do not have an alternative but we should add something useful here to note why it is deprecated:
```java
@deprecated parent {@link AbstractLinkedList} is source incompatible with List methods added in Java 21
```
If a request comes to add a replacement then it can be done under a new enhancement.